content/en/docs/FAQ/installation.md

```diff
@@ -3,6 +3,14 @@ title: "Installation"
 description: "Questions about Kiali installation options or issues."
 ---
 
+### What is the difference between the operator and the server helm chart?
+
+There are two installation mechanisms from which you can choose when installing the Kiali Server. The first is the recommended installation mechanism - the Kiali Operator. The second installation mechanism is the server helm chart. There are some features that you get with the Kiali Operator that you do not get with the server helm chart. The main differences between the two are mentioned below, though this list may be incomplete.
+
+1. The operator watches for changes to the multi-cluster remote cluster secret - if a change is detected, the operator automatically rolls out a new Kiali server pod so the server picks up the changes immediately. See the [multi-cluster docs](/docs/configuration/multi-cluster/#setup) for more details.
```

Comment:
Isn't it true that the operator, in general, also watches for CR changes and automatically rolls out a new Kiali server pod?

Reply:
Yes, but I didn't want to be verbose here talking about basic functionality differences between general helm charts and operators. I wanted to concentrate on explaining where feature parity was "broken" specifically between the kiali server helm chart and the kiali operator.

Reply:
In other words, look at the settings in the Kiali CR - what settings in there behave differently (or don't work at all) when using the server helm chart? That's what this FAQ is addressing. Perhaps we can be more explicit saying that here.

Reply:
OK.